torch/csrc/cuda/Module.cpp

```diff
@@ -136,7 +136,7 @@ PyObject * THCPModule_getRNGState(PyObject *_unused)
   using namespace torch::autograd;
   HANDLE_TH_ERRORS
   Variable var = torch::empty(0, at::device(at::kCPU).dtype(at::kByte));
-  THCRandom_getRNGState(state, (THByteTensor*)(var.data().unsafeGetTensorImpl()));
+  THCRandom_getRNGState(state, (THByteTensor*)(var.unsafeGetTensorImpl()));
```

Comment:
The original use of `.data()` here is to make sure that the `THCRandom_getRNGState(state, ...)` call is using the Variable `var`'s underlying `data_`'s TensorImpl instead of its `Variable::Impl`. Since now there is no `data_` and a Variable stores its tensor data directly in its TensorImpl, and the `THCRandom_getRNGState(...)` call doesn't care about whether the TensorImpl has autograd metadata, it's okay to remove `.data()` here.